econml/inference.py

```diff
@@ -1147,12 +1164,17 @@ def conf_int_point(self):
             the corresponding singleton dimensions in the output will be collapsed
             (e.g. if both are vectors, then the output of this method will also be a vector)
         """
-        lower_ci_point = np.array([self._mixture_ppf(self.alpha / 2, self.pred, self.pred_stderr, self.tol)])
-        upper_ci_point = np.array([self._mixture_ppf(1 - self.alpha / 2, self.pred, self.pred_stderr, self.tol)])
-        return np.array([lower_ci_point]) if np.isscalar(lower_ci_point) else lower_ci_point,\
-            np.array([upper_ci_point]) if np.isscalar(upper_ci_point) else upper_ci_point
+        alpha = self.alpha if alpha is None else alpha
+        tol = self.tol if tol is None else tol
+        lower_ci_point = np.array([self._mixture_ppf(alpha / 2, self.pred, self.pred_stderr, tol)])
+        upper_ci_point = np.array([self._mixture_ppf(1 - alpha / 2, self.pred, self.pred_stderr, tol)])
+        return lower_ci_point, upper_ci_point
 
-    def print(self):
+    def summary(self, alpha=0.1, value=0, decimals=3, tol=0.001, output_names=None, treatment_names=None):
```

Comment:
Why do we need both `summary` and `print`, which are the same except for their default values?

If this method is really needed, please add a docstring.

Reply:
good point, maybe we can merge. My thought: print is an internal api method and summary public facing. Maybe if I change print to `_print` it might be better.

Reply:
I moved it to a private helper function. We do need two functions, vecause we want to use them with different defaults